CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 FEATURES:
 * `d/tfe_registry_module`: Add `vcs_repo.tags` and `vcs_repo.branch` attributes to allow configuration of `publishing_mechanism`. Add `test_config` to support running tests on `branch`-based registry modules, by @hashimoon [1096](https://github.com/hashicorp/terraform-provider-tfe/pull/1096)
+* **New Resource**: `r/tfe_organization_default_execution_mode` is a new resource to set the `default_execution_mode` and `default_agent_pool_id` for an organization, by @SwiftEngineer [1137](https://github.com/hashicorp/terraform-provider-tfe/pull/1137)'
+* `r/tfe_workspace`: Now uses the organization's `default_execution_mode` and `default_agent_pool_id` as the default `execution_mode`, by @SwiftEngineer [1137](https://github.com/hashicorp/terraform-provider-tfe/pull/1137)'
```

Comment:
I don't think this breaks the API of the provider or the API itself because it's a purely additive change. If you don't change the default at the org level, everything continues to behave as before.

Reply:
> If you don't change the default at the org level

Yeah but what if someone does? 🙂 
The provider is still pre-1.0 so everything is allowed. I'm just suggesting to add a specific note about this in the changelog.

Reply:
That's a fair point. Previously, I was thinking setting the default `execution_mode` in an admission that you were prepared for that change to affect all the workspaces in your organization, but adding a note here couldn't hurt!